### ISSUE TITLE
Test gradle plugin works on Windows

### ISSUE THREAD

**Issue by levibostian** [levibostian/dotenv-android#14]
Just like [this issue](https://github.com/levibostian/dotenv-android/issues/13) except testing on the Windows OS. 

# How? 

1. Expand the GitHub Actions test matrix to include Windows OS (x86, x64, arm?)
2. On GitHub Actions, download and install the `dotenv` CLI binary to the CLI machine. 
3. Run `./gradlew assembleDebug` for the Android app and see if the app successfully builds. 

Note: We may need to use the gradlew batch file instead? I have never used the command line for compiling Android apps on Windows.

**Comment by jayjacksonworks** [comment]
Hey Levi, I just came across this open issue. One of my developers is using a Windows machine and he mentioned that we was having issues getting his Android project to build using this library. It looks like some of the dotenv-android dependencies do not work for Windows.

I just wanted to confirm that this library only supports/tested on Unix or if there are special instructions for Windows install (https://github.com/levibostian/dotenv#install)?

Although our project is a React Native project, his full stacktrace is below: 
```

WARNING:: Please remove usages of `jcenter()` Maven repository from your build scripts and migrate your build to other Maven repositories.
This repository is deprecated and it will be shut down in the future.
See http://developer.android.com/r/tools/jcenter-end-of-service for more information.
Currently detected usages in: root project 'XXXX', project ':amplitude_react-native', project ':app', ...

> Task :app:generateDebugDotenv FAILED
=== dotenv-android plugin running ===
dotenv-android info:
dotenv CLI compatible version: >= 1.0.0 && < 2
downloading CLI dependency...
It appears you are using Windows as your operating system. Sorry, but at this time dotenv-android requires that you manually install a dependency in order to use this Gradle plugin.
ReactNativeFirebase WARNING: NPM package '@react-native-firebase/analytics' depends on '@react-native-firebase/app' v11.3.0 but found v11.5.0, this might cause build issues or runtime c
rashes.
ReactNativeFirebase WARNING: NPM package '@react-native-firebase/auth' depends on '@react-native-firebase/app' v12.1.0 but found v11.5.0, this might cause build issues or runtime crashe
s.
ReactNativeFirebase WARNING: NPM package '@react-native-firebase/crashlytics' depends on '@react-native-firebase/app' v11.3.3 but found v11.5.0, this might cause build issues or runtime
 crashes.
ReactNativeFirebase WARNING: NPM package '@react-native-firebase/firestore' depends on '@react-native-firebase/app' v11.3.0 but found v11.5.0, this might cause build issues or runtime c
rashes.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings
12 actionable tasks: 2 executed, 10 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:generateDebugDotenv'.
> Follow the install instructions: https://github.com/levibostian/dotenv#install and try running the plugin again.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 39s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:generateDebugDotenv'.
> Follow the install instructions: https://github.com/levibostian/dotenv#install and try running the plugin again.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

**Comment by levibostian** [comment]
The [dotenv cli](https://github.com/levibostian/dotenv) project that powers the `dotenv-android` gradle plugin is simply a binary program meaning you can [download and install any of the pre-compiled](https://github.com/levibostian/dotenv/releases/tag/1.1.0) files for your OS to run the tool. As long as the gradle plugin can find the CLI executable, the gradle plugin will run it and everything should work. 

Everything *might* work on Windows but the only problem is that I have never tested it before. I have only used both `dotenv` CLI and `dotenv-android` with macOS and Linux users so Windows support is not officially supported simply because it has not yet been tested. Although because you see I created this issue, I want Windows support to exist. 

The error message `It appears you are using Windows as your operating system. Sorry, but at this time dotenv-android requires that you manually install a dependency in order to use this Gradle plugin.` is the CLI telling you that yes you must manually install the `dotenv` CLI tool instead of the default behavior of the `dotenv-android` *gradle plugin downloading and installing it for you*. 

If your Windows developer was to [download a binary for their Windows OS](https://github.com/levibostian/dotenv/releases/tag/1.1.0) and add the CLI to their PATH so that you can simply execute `dotenv` and it points to an executable then this plugin *might* work as expected. 

In conclusion, I would like to get Windows to work with this gradle plugin project. The plugin has been developed to try to make it work, however, it has never been tested so there are no guarantees. If you are looking to move forward with your project and not hit any roadblocks, I recommend that you find another way to develop your Android app with your team member on Windows not using this gradle plugin. However, if you would like to help me out on this project by trying things on Windows and showing me debug error logs I would love to add Windows support!

**Comment by jayjacksonworks** [comment]
Thanks for the response, we'll give this a try and get back to you.
